docs/data-import.rst

```diff
@@ -19,17 +21,25 @@ To load the database dump, run:
     unzip traffic_stops_nc_production_20150801.tar.zip
     pg_restore -Ox -d traffic_stops_nc traffic_stops_nc_production.tar
 
-To create the database dump, run:
+Browse https://s3-us-west-2.amazonaws.com/openpolicingdata/ to see what dumps
```

Comment:
FWIW @copelco hasn't stored any db dumps on S3 for a while AFAICT; perhaps that should be one of the end-of-project/go-live tasks